addons/crm_iap_lead/models/crm_iap_lead_mining_request.py

```diff
@@ -242,22 +242,58 @@ def action_submit(self):
 
     def action_get_lead_action(self):
         self.ensure_one()
+        try_again_action_id = self.env.ref('crm_iap_lead.action_crm_iap_lead_mining_request_try_again').id
```

Comment:
Should probably handle action being deleted by user. No need to crash if it has been removed.